extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -136,32 +119,32 @@ describes.realWin('Bind', {
   it('should throw error if experiment is not enabled', () => {
     toggleExperiment(env.win, 'amp-bind', false);
     // Experiment check is bypassed on test mode -- make sure it isn't.
-    window.AMP_MODE = {test: false};
+    env.sandbox.stub(getMode(), 'test', false);
```

Comment:
Use `AMP_MODE`, we exported it specifically for this. You can use `Object.assign` if you're worried about conflicting properties.